notes/inheritance.js

```diff
@@ -0,0 +1,135 @@
+/**
```

Comment:
This is really really fun! I'd like to rephrase something though. Objects in Javascript don't necessarily "Inherit" they way that you may be used to from other OO Languages. It's more that a "Parent" class would be "Delegating" do it's "Sub-classes". I'm sure you're aware of these intricacies, but I wanted to make it very clear [Here](http://www.datchley.name/understanding-prototypes-delegation-composition/) is a link to a great article that talks about Prototypal 'inheritance' which really should say prototypal 'delegation'